staging/src/k8s.io/client-go/tools/remotecommand/v4_test.go

```diff
@@ -31,6 +31,10 @@ func TestV4ErrorDecoder(t *testing.T) {
 	}
 
 	for _, test := range []Test{
+		{
+			message: "",
+			err:     "error stream protocol error: unexpected end of JSON input in \"\"",
```

Comment:
this is a really confusing message for v4 to return when the real problem is that no completion status was returned over the error stream... can we improve this?